src/main/java/uk/gov/hmcts/cmc/claimstore/services/DirectionsQuestionnaireService.java

```diff
@@ -45,7 +46,7 @@ public CaseEvent prepareCaseEvent(ResponseRejection responseRejection, Claim cla
             return REFERRED_TO_MEDIATION;
         }
 
-        if (!isOnlineDQ(claim)) {
+        if (!isOnlineDQ(claim) && ClaimantResponseHelper.hasNotOptedForMediation(responseRejection)) {
```

Comment:
Is this clause needed. The previous if already checks that mediation has not been opted for.

Reply:
not required